akka-docs/src/test/scala/docs/stream/operators/sourceorflow/GroupedWeighted.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sourceorflow
+import akka.stream.scaladsl.Source
+import scala.collection.immutable
+
+object GroupedWeighted {
+  def groupedWeightedExample(): Unit = {
+    import akka.actor.ActorSystem
+
+    implicit val system: ActorSystem = ActorSystem()
+
+    //#groupedWeighted
+    val collections = immutable.Iterable(Seq(1, 2), Seq(3, 4), Seq(5, 6))
+    Source[Seq[Int]](collections).groupedWeighted(4)(_.length).runForeach(println)
+    // Vector(Seq(1, 2), Seq(3, 4))
+    // Vector(Seq(5, 6))
+
+    Source[Seq[Int]](collections).groupedWeighted(3)(_.length).map(_.flatten).runForeach(println)
+    // Vector(1, 2, 3, 4)
+    // Vector(5, 6)
```

Comment:
I think it would be better to skip the extra brain gymnastics with the map(flatten) here and just show exactly what comes out downstream, just like the sample above